.github/tools/containerAppJobVerifier.sh

```
@@ -1,3 +1,9 @@
+if ! command -v jq 2>&1 >/dev/null
```

Shellcheck

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)